### DISCUSSION TITLE
Selecting entries in contents list highlights entry above

### DISCUSSION THREAD

**Discussion by svksn** [executablebooks/meta#814] [Q&A]
Hi, 

I have a book compiled with v0.13.1. When navigating with the content list on the right hand side, the link jumps to the correct section. But in the contents list, the entry above the one clicked is highlighted, not the one that I just jumped to. If I click the topmost section, none is highlighted.

Any ideas what I might be doing wrong or where I should start looking?

**Comment by choldgraf** [comment]
Yep - this one is a bug in the book theme that is fixed upstream but we still need to finish updating to the latest version of the pydata theme.

See issue about it here:

- https://github.com/executablebooks/sphinx-book-theme/issues/607
